docs/concepts/what-is-a-schedule.md

```diff
@@ -0,0 +1,197 @@
+
+Note that with the `AllowAll` overlap policy, this pause might not apply to the next Run, because the next run might have started before the failed run finished.
+It applies only to Runs that were Scheduled to start after the failed Run finished.
+
+### Last completion result
+
+A Workflow started by a Schedule can obtain the completion result from the most recent successful run. (How you do this depends on which SDK you're using.)
+
+For overlap policies that don't allow overlap, “the most recent successful run” is straightforward to define.
+For the `AllowAll` policy, it refers to the run that completed most recently, at the time that the run in question is started.
+That is, given the overlapping runs below:
+
+```
+time -------------------------------------------->
+ A     |----------------------|
+ B               |-------|
+ C                          |---------------|
+ D                                |--------------T
+```
+
+If D asks for the last completion result at time T, it will get the result of A.
+Not B, even though B started more recently, because A completed later. And not C, even though C completed after A, because the result for D is captured when D is started, not when it's queried.
+
+Failures and timeouts do not affect the last completion result.
+
+### Last failure
+
+A Workflow started by a Schedule can obtain the details of the failure of the most recent run that ended at the time when the Workflow in question was started. Unlike last completion result, a _successful_ run _does_ reset the last failure.
+
+### How to enable Schedules
+
+:::info Experimental
+
+The Scheduled Workflows feature is available in Temporal Server version 1.17.
+However the feature is in an experimental stage and is disabled by default.
+
+Internally, a Schedule is implemented as a Workflow.
+These implementation Workflow Executions are visible to you as you navigate the Web UI and use tctl, though you should not interact with it directly.
+
+In later versions the implementation Workflows will cease to be visible by default.
+
+:::
+
+**Requirements**
+
+- Temporal Server version 1.17 or later.
+- [Advanced Visibility](/concepts/what-is-advanced-visibility) optional.
+- The following dynamic config values:
+
+```yaml
+frontend.enableSchedules:
+  - value: true
+worker.enableScheduler:
+  - value: true
+matching.numTaskqueueReadPartitions:
+  - value: 1
+    constraints:
+      taskQueueName: temporal-sys-scheduler-tq
+matching.numTaskqueueWritePartitions:
+  - value: 1
+    constraints:
+      taskQueueName: temporal-sys-scheduler-tq
+```
+
+Only the first two are required; the second two are suggested because by default, only one Worker will run per Task Queue, so more than one partition is not useful.
+Setting the Task Queue to use one partition will reduce latency.
+
+If you're familiar with Dynamic Config, you may also constrain these settings by Namespace, as makes sense for your installation.
```

Comment:
```suggestion
Only the first two values are required; the second two are suggested because, by default, only one Worker runs per Task Queue, so more than one partition is not useful.
Setting the Task Queue to use one partition reduces latency.

If you're familiar with Dynamic Config, you can also constrain these settings per Namespace as needed for your installation.
```